modules/intro/pages/transaction-and-acid.adoc

```diff
@@ -43,12 +47,19 @@ Multiple reads in a single read-only transaction T1 will get the same results, e
 [discrete]
 ==== *Durability*  +++<a id="TigerGraphTransactionProcessingandACIDPropertiesSupport-Durability">++++++</a>+++
 
-Committed transactions are written to disk (SSD or HDD). The TigerGraph platform implements write-ahead logging (WAL) to provide durability.
+Committed transactions are written to disk. The TigerGraph platform implements write-ahead logging (WAL) to provide durability.
```

Comment:
I think you should leave the parenthetical phrase. It answers implied questions about what sort of disks does TigerGraph support, in a concise and unobtrusive way (in my opinion).